api/src/main/java/ai/djl/modality/cv/translator/YoloV5Translator.java

```diff
@@ -150,26 +149,27 @@ protected double overlap(double x1, double w1, double x2, double w2) {
     }
 
     private DetectedObjects processFromBoxOutput(NDList list) {
-        NDArray ndArray = list.get(0);
+        float[] flattened = list.get(0).flatten().toFloatArray();
```

Comment:
You can just call toFloatArray() if what you want is 1-D data without the shape information
```suggestion
        float[] flattened = list.get(0).toFloatArray();
```